packages/react-core/src/components/Menu/MenuItem.tsx

```diff
@@ -295,6 +298,7 @@ const MenuItemBase: React.FunctionComponent<MenuItemProps> = ({
         isLoadButton && styles.modifiers.load,
         isLoading && styles.modifiers.loading,
         isFocused && styles.modifiers.focus,
+        isDanger && styles.modifiers.danger,
```

Comment:
Typically for these statuses, we include some sr-only text https://github.com/patternfly/patternfly-react/blob/60f9853c6617adbc51dfd015f03a731c5141b421/packages/react-core/src/components/Popover/Popover.tsx#L452

We have the sr-only text in the core example. Should we add that? cc @thatblindgeye

Reply:
This feels like an interesting scenario to me. I think it could really go either way, but it would bring into question whether the "danger"/"warning" variant of our Button component should have similar visually hidden text.

I don't believe I've seen visually hidden text like this used for "danger" menu items/buttons before, and it could possibly stray into being too verbose (especially the more visually hidden text there is on a page)... but at the same time, if we want to visually convey these menu items as "danger" items to users that are sighted, regardless if there's an additional step from a confirmation modal, it could make sense to convey that a menu item is a "danger" item to users of AT similarly.

Looking at Web-AIM's [invisible content](https://webaim.org/techniques/css/invisiblecontent/) page in the "Examples" section, they have a note about:

>Use these techniques judiciously! Keep in mind that many screen reader users have some vision—what they see and what they hear should typically be in harmony. In general, screen reader-only content should be reserved for information is apparent visually but not apparent to blind screen reader users.

Which this scenario could technically fit into that sort of criteria (a "danger" item is apparent visually via the text color, but not as such to users that are blind and navigating via SR).

It could possibly come down to whether the text of the "danger" menu item is enough to convey that it is a dangerous/destructive action (i.e. will these dangerous items always be as clear as "Delete"), or whether the additional context via color and/or visually hidden text is needed to help convey that dangerous/destruction nature (i.e. is the text color more to convey an item as dangerous/destructive, or more to just visually separate it from other menu items?).

I think I'd generally be okay with SR text being added, but it would probably lead to a broader convo regarding other components like Button that also have status/severity sort of visual styling.